presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -59,6 +59,7 @@
 import com.facebook.presto.operator.scalar.ArrayGreaterThanOperator;
```

Comment:
Yes, I will add some comments in the commit message. The commit order looks wrong, but it looks good with `git log`, I used `git rebase -i` to do some re-order, but Github seems to list the commits by their creation time. I will also fix it.